src/kibana/components/vislib/visualizations/area_chart.js

```diff
@@ -161,12 +162,12 @@ define(function (require) {
         .append('g')
         .attr('class', 'points area');
 
-      // append the bars
+      // append the circles
       circles = layer
-      .selectAll('rect')
+      .selectAll('circles')
       .data(function appendData(data) {
         return data.filter(function isNotZero(d) {
```

Comment:
Probably shouldn't call this `isNotZero` anymore, since it's checking for zero or null now